src/chrome/content/rules/Rubicon-Project.xml

```diff
@@ -15,53 +15,134 @@
 	* Plaintext reply
 
 
-	Insecure cookies are set for these domains:
-
-		- .rubiconproject.com
+	Timeout:
+		- 204.rubiconproject.com
+		- arws.rubiconproject.com
+		- bcbb-hfc2.rubiconproject.com
+		- bcbb-iad2.rubiconproject.com
+		- boxy-bea-iad2.rubiconproject.com
+		- content.rubiconproject.com
+		- pas-api.rubiconproject.com
+		- vpn-it.rubiconproject.com
+
+	Cert expired:
+		- gobuyersupport.rubiconproject.com
+		- sonic.rubiconproject.com
+
+	Chain issues:
+		- vpn-dc.rubiconproject.com
+		- vpn-eng.rubiconproject.com
+		- vpn-test.rubiconproject.com
+		- vpn.rubiconproject.com
+
+	Connection refused:
+		- rfm.rubiconproject.com
+	
+	Cert mismatch:
+		- awsm.rubiconproject.com
+		- dsp-token.aws.rubiconproject.com
+		- assets.rubiconproject.com
+		- assets2.rubiconproject.com
+		- cdn.rubiconproject.com
+		- email.rubiconproject.com
+		- garagestaging.rubiconproject.com
+		- go.rubiconproject.com
+		- investor.rubiconproject.com
+		- labs2.rubiconproject.com
+		- messages.rubiconproject.com
+		- assets.rfm.rubiconproject.com
+		- rubibid.rubiconproject.com
+		- stats.rubiconproject.com
+		- status.rubiconproject.com
+		- tap-cdn.rubiconproject.com
+	
+	Non-200 status code:
+		- tap2-cdn.rubiconproject.com
 
 -->
 <ruleset name="Rubicon Project.com">
 
-	<!--	Direct rewrites:
-				-->
-	<target host="ads.rubiconproject.com" />
+	<target host="rubiconproject.com" />
+	<target host="www.rubiconproject.com" />
```

Comment:
MCB

Reply:
maybe we should enable Upgrade Insecure Requests outside of the block http ressources mode

Reply:
@J0WI can you check that again? The only thing i see browsing a little bit there is a failed geo lookup due to CORS from teletext.io, so obvious breakage.